docling/models/page_assemble_model.py

```diff
@@ -52,6 +52,14 @@ def sanitize_text(self, lines):
 
         sanitized_text = "".join(lines)
 
+        # Text normalization
```

Comment:
This looks dangerous ... should we try to have this separated?

Reply:
It is separated already. This is the central method that takes care of making text items.